src/chromo/models/dpmjetIII.py

```diff
@@ -213,10 +213,11 @@ def _generate(self):
 
 class DpmjetIII191(DpmjetIIIRun):
     _version = "19.1"
+    _projectiles = standard_projectiles | Nuclei() | {3322, 3312, 3222, 3122, 3112, 311}
```

Comment:
If it is shared between DpmjetIII191 and  DpmjetIII193, it should probably be in DpmjetIIIRun. This also works, but putting common code there is cleaner.